packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -1,80 +1,91 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name rubyipmi
 
-Summary: A ruby wrapper for ipmi command line tools
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.10.0
-Release: 3%{?dist}
+Release: 4%{?dist}
+Summary: A ruby wrapper for ipmi command line tools that supports ipmitool and freeipmi
 Group: Development/Languages
-License: LGPLv2
-URL: https://github.com/logicminds/rubyipmi
+License: LGPLv2.1
+URL: http://github.com/logicminds/rubyipmi
```

Comment:
this is pulled from the gem metadata, but I think we should "correct" it here. (I've opened a PR upstream to fix that: https://github.com/logicminds/rubyipmi/pull/49)